src/stores/RoomViewStore.js

```diff
@@ -224,6 +215,7 @@ class RoomViewStore extends Store {
                     storeRoomAliasInCache(payload.room_alias, result.room_id);
                     roomId = result.room_id;
                 } catch (err) {
+                    console.error(err);
```

Comment:
debug line that can be removed?

Reply:
I was intending it as a way to actually put the error into the rageshakes, will add context.